src/tests/Dashboard/container/ConceptModal.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import AddConceptModal from '../../../components/dashboard/components/concepts/ConceptModal';
+
+describe('Concept Modal', () => {
+  it('should render without crushing', () => {
+    const wrapper = mount(<AddConceptModal />);
+    expect(wrapper).toMatchSnapshot();
+  });
```

Comment:
You could actually do more assertions here

Reply:
Ideas are welcome but for now, this is enough since its just a component without logic